lib/sass_spec/runner.rb

```diff
@@ -42,24 +42,7 @@ def run
       Minitest.reporter = Minitap::TapY
     end
 
-    result = Minitest.run(minioptions)
-
-    if @options[:run_todo]
```

Comment:
It's a big pain in the ass to support, so it might be worth altering workflows for. You can still find passing TODO tests using `--interactive --probe-todo`.